### PR TITLE
Add health appender

### PR DESCRIPTION
**Pull request by nrfox**
Backend PR: https://github.com/kiali/kiali/pull/4802

### CODE REVIEW

File: src/services/GraphDataSource.ts [hunk 1]
```diff
@@ -182,7 +182,7 @@ export default class GraphDataSource {
     }
 
     // Some appenders are expensive so only specify an appender if needed.
-    let appenders: AppenderString = 'deadNode,istio,healthConfig,serviceEntry,sidecarsCheck,workloadEntry';
+    let appenders: AppenderString = 'deadNode,istio,healthConfig,serviceEntry,sidecarsCheck,workloadEntry,health';
```
Comment (nrfox) [inline src/services/GraphDataSource.ts:185]: Will probably change this to something like `nodeHealth` or `graphHealth` in final version

Comment (jshaughn) [reply src/services/GraphDataSource.ts:185]: This is likely going to change given what I did in https://github.com/israel-hdez/swscore/pull/23.  That sub-pr already introduces a new `health` appender (finalizer), and merges the healthConfig appender into it.

**Review by jshaughn — CHANGES_REQUESTED** [review]

**Comment by aljesusg** [comment]
This PR must be submit to https://github.com/kiali/kiali

**Comment by lucasponce** [comment]
I'm going to close it to avoid confusion.
Please @nick, would you mind to re-arrange the work to target the unified repo?
Thanks

**Comment by nrfox** [comment]
> I'm going to close it to avoid confusion.
> Please @nick, would you mind to re-arrange the work to target the unified repo?
> Thanks

That is not my username :) but it is fine that this issue is closed. These changes are already covered by previous work on graph health: https://github.com/kiali/kiali-ui/pull/2316.

**Comment by lucasponce** [comment]
> That is not my username :)

Ouch, sorry, Then there will be another Nick with a lot of notifications from my side.

**Pull request closed without merging**